packages/firestore/test/integration/api/aggregation.test.ts

```diff
@@ -38,4 +39,23 @@ apiDescribe('Aggregation query', (persistence: boolean) => {
       expect(snapshot.getCount()).to.equal(3);
     });
   });
+
+  it('getAggregateFromServerDirect fails if user is offline async', () => {
+    const testDocs = {
```

Comment:
Can we just use an empty collection here? I don't think it matters how many documents are present in the collection, and, if that is the case, then populating the collection somewhat implies that the contents of the collection are meaningful.